subjects/hello-world/README.md

```diff
@@ -0,0 +1,43 @@
+# hello-world
+
+| Complexity | Executable | Goal |
+|------|------|------|
+| Low | `greet` | Create a CLI Program that outputs a greeting.
+
+This subject is purely aimed to trying out the validation workflow.
+
+## Feature
+
+### Greet
+
+The program `greet` *may* have an argument to specify who to greet.
+
+```console
+$ greet Bob
+Hello Bob.
+```
+
+If no person to greet is specified, `greet` the World.
+
+```console
+$ greet
+Hello World.
+```
+
```

Comment:
And if you enter more than 1 arguments, will the program greet only the first one, or all of them ?

Reply:
I find simply considering every arguments as a single argument for this one as the exercise is solely here to try out the validation process :)

I'll add an example to cover this kind of questions

Reply:
Should be covered by c657037 let met know what you think !

Reply:
Very interesting, I find this more relevant ! :)